scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -587,10 +592,50 @@ <h6>No pinned variants</h6>
         </li>
       {% endfor %}
     </ul>
-    </div>
+    {{ clinvar_vars_summary(suspects, case, institute) }}
+      </div>
   </div>
 {% endmacro %}
 
+{% macro clinvar_var_status(variant, case, institute) %}
+    {% if variant._id and variant.category != 'str' %}
+        <form id="clinvar_submit" class="d-flex justify-content-center" action="{{ url_for('clinvar.clinvar_add_variant', institute_id=institute._id, case_name=case.display_name) }}" method="POST">
+            {% if case.clinvar_variants and variant._id in case.clinvar_variants.keys() %}
+                (included in ClinVar submission)
+            {% else %}
+                <button type="submit" name="var_id" value="{{variant._id}}" class="btn btn-secondary btn-sm" style="float: right;">Add to ClinVar submission</button>
+            {% endif %}
+        </form>
+    {% endif %}
+{% endmacro %}
+
+{% macro clinvar_vars_summary(suspects, case, institute) %}
```

Comment:
This looks some complicated code, would be much better to move it to the backend instead.

Reply:
I have moved it to the controller. I ended up deciding to keep the case.clinvar_variants for the counter and only adding the unpinned variants included in submission in  case_obj["clinvar_variants_not_in_suspects"] and using this to render the pretty_link

Reply:
I think it is good enough to merge as it is. Also because if you move the button on the side and there are no variants in ClinVar that are are no longer pinned it would look asymmetric. Keep it as it is!